src/test/ui/privacy/public-instead-of-pub.rs

```diff
@@ -0,0 +1,4 @@
+// Checks what happens when `public` is used instead of the correct, `pub`
+// edition:2018
```

Comment:
```suggestion
// edition:2018
// run-rustfix
```

We should test that applying the suggestion works too.